pghoard/pgutil.py

```diff
@@ -92,3 +93,15 @@ def parse_connection_string_libpq(connection_string):
                 value, connection_string = rem, ""
         fields[key] = value
     return fields
+
+
+def check_if_pg_connection_is_alive(db_conn) -> bool:
```

Comment:
This is a bit heavy, may be better to check the connection is not closed.
Since we are the only one to use it, it should be enough, but additionally one can check the transaction status.
 
 See how the psycopg2 builtin pooling implements this: 
 https://github.com/psycopg/psycopg2/blob/9e6c3322d8640bca7007a222973d87d8ea60057c/lib/pool.py#L103
 
 As well as how the libpq reports transaction status: 
 https://www.postgresql.org/docs/current/libpq-status.html#LIBPQ-PQTRANSACTIONSTATUS

Reply:
Ah okay, makes sense. I'll check if both `db_conn.closed is False` and if the transaction status is not unknown. I mean for this case we are only interested if we lost connection